image-customize

```diff
@@ -258,7 +268,7 @@ def main():
     machine = testvm.VirtMachine(maintain=True,
                                  verbose=args.verbose,
                                  networking=network.host(restrict=args.no_network),
-                                 image=prepare_install_image(args.base_image, args.image, args.resize))
+                                 image=prepare_install_image(args.base_image, args.image, args.resize, args.fresh))
```

Comment:
Another good time to mention that I enjoy just passing the entire `args` through to the various action functions... :)

[note: not a change request!]

Reply:
Heh, matter of style -- I actually prefer function arguments to be explicit, and not couple them to the argparse structure.